### DISCUSSION TITLE
Re-pairing not necessary?

### DISCUSSION THREAD

**Discussion by Stooovie** [Koenkk/zigbee2mqtt#18869] [General]
Hi, I had to change the channel trying to troubleshoot some interference and to my surprise, all 40 devices just came back online. And yes I did it properly:
- shut down Z2M
- unplug hub (CC2562RB) for 10m
- change channel from 11 to 20
- Plug back in and start Z2M

Not complaining or anything, just curious if this is even possible.

**Comment by Stooovie** [comment]
Heh, no, it's just that Z2M IGNORES the channel setting in the addon yaml, you can only set it in Z2M UI.

**Comment by Daniel76145** [comment]
As per note in zigbee2mqtt it says:
_Zigbee channel, changing requires repairing all devices! (Note: use a ZLL channel: 11, 15, 20, or 25 to avoid Problems)_
I am supprised that you did not have to repair all devices again.
